components/globals/Base.js

```diff
@@ -7,9 +7,29 @@ import SkipLink from "./SkipLink";
 import Fip from "./Fip";
 
 const Base = ({ children }) => {
+  const isProduction = process.env.NODE_ENV === "production";
   return (
     <>
       <Head>
+        {isProduction && (
+          <React.Fragment>
+            <script
+              async
+              src="https://www.googletagmanager.com/gtag/js?id=G-KY2EVJV33K"
+            ></script>
+            <script
+              dangerouslySetInnerHTML={{
```

Comment:
I can approve the PR as is, but was wondering if it's worth iterating without `dangerouslySetInnerHTML` via something like this (not installing the plugin, but adding a dom node?) https://github.com/alinemorelli/react-gtm/blob/master/src/TagManager.js#L41

Also, I remember something funky about client-side routing and maybe we don't need to tinker in the router to but rather set a "Google Tag Manager's History Change trigger": https://support.google.com/tagmanager/answer/7679322?hl=en

Happy to merge this for the demo and iterate later, let me know

Reply:
`dangerouslySetInnterHTML` is Reacts way of wrapping the example you provided instead of creating a separate DOM node and then injecting it.  [React DOM dangerouslySetInnerHTML](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml).  It basically just saves lines of code but if you want to iterate it out and inject it as a pure DOM we can do that to.

Reply:
We're currently not using Google Tag manager. (But we can setup if needed.) Since everything is SSR right now I was curious to see if all states would be currently handled by the base script.  Once we know from Research what they want to track I think it would be good at that point to ensure everything's captured?

Reply:
Sounds good, let's merge this and add a follow-up card for PM/Research to test and let us know if anything needs to be tweaked.